### DISCUSSION TITLE
Kor's Object in ubuntu

### DISCUSSION THREAD

**Discussion by shividwivedi** [eyurtsev/kor#321] [Q&A]
I am running code using kor's Object with async and it is running without any error the windows but on ubuntu it is giving me problem that 'Object' object is not subscriptable.

**Comment by shividwivedi** [comment]
Does any one have answer regarding this!.I am only using Kor's obejct for making schema and it is running fine on windows but on ubuntu i am getting error

**Reply by eyurtsev** [reply]
Hi @shividwivedi, please take a look at these guidelines for asking debugging questions: https://stackoverflow.com/help/minimal-reproducible-example

**Reply by shividwivedi** [reply]
okay sure  i will check and from next time i will make sure to ask questions in the given format only , Thank you for sharing and  issue is resolved.
